src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -212,7 +213,6 @@ const pageQuery = gql`
 				}
 				geocode {
 					city
-					state
```

Comment:
Is this not needed for comparison in @eddieferrer 's new background image work?

Reply:
`HeroBackground` has it's own query.